tests/generate.ts

```diff
@@ -104,6 +104,25 @@ testProcessProject(
   }
 )
 
+testProcessProject(
+  'generates type guards for empty object if exportAll is true',
+  {
+    'test.ts': `
+    export interface Empty {}`,
+  },
+  {
+    'test.guard.ts': `
+    import { Empty } from "./test";
+
+    export function isEmpty(obj: any, _argumentName?: string): obj is Empty {
+        return (
+            typeof obj === "object"
```

Comment:
Not relevant to this feature, but just FYI this check has always been wrong: #60